sysdump/sysdump.go

```diff
@@ -2123,6 +2164,46 @@ func (c *Collector) submitKVStoreTasks(ctx context.Context, pod *corev1.Pod) err
 	return nil
 }
 
+// submitMetricsSubtask submits tasks to collect metrics from pods.
+func (c *Collector) submitMetricsSubtask(pods *corev1.PodList, containerName, portName, filenameTmpl string) error {
+	for _, p := range pods.Items {
+		p := p
+		if p.Status.Phase == "Running" {
```

Comment:
nit: We should use the predefined constant here for the pod phase i.e v1.PodRunning.

Reply:
nit: Also we could considere short circuiting the loop here since we only care about running pods to keep the indent to a minimum.